integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -198,21 +203,28 @@ public void addStopHook(Callable<? extends CompletionStage<?>> hook) {
             }
         }, actorSystem.dispatcher());
 
+        ServiceInfo serviceInfo = new ServiceInfo(serviceName);
+
+        // Kafka client
+        TopicFactory kafkaTopicFactory = new KafkaTopicFactory(serviceInfo, actorSystem, materializer,
+                actorSystem.dispatcher());
+        TopicFactoryProvider topicFactoryProvider = () -> Some.apply(kafkaTopicFactory);
```

Comment:
Does this mean that use of this class assumes that Kafka is in use? Shouldn't it be configurable?